### PR TITLE
fix: update the error message when git is missing.

### PR DESCRIPTION
**Pull request by raksiv**

### CODE REVIEW

File: pkg/templates/contents.go [hunk 1]
```diff
@@ -134,7 +134,7 @@ func (d *downloader) repository() error {
 
 	// download file
 	if err := client.Get(); err != nil {
-		return fmt.Errorf("error getting path %s: %w", src, err)
+		return fmt.Errorf("Nitric relies on Git to fetch dependencies. Please refer to the installation instructions - https://nitric.io/docs/installation")
```
Comment (asalkeld) [inline pkg/templates/contents.go]: I don't think this uses git at all, it does an http download of the git repo. Same below

Comment (raksiv) [reply pkg/templates/contents.go]: getter is unique in its ability to download both directories and files. It also detects certain source strings to be protocol-specific URLs. For example, "github.com/hashicorp/go-getter" would turn into a Git URL and use the Git protocol.

Comment (raksiv) [reply pkg/templates/contents.go]: Can you double check? I'm pretty sure this is using git - 

The error the I received was directly as a consequence of removing git from my machine.

Comment (asalkeld) [reply pkg/templates/contents.go]: I think it would be good to check for the particular error, otherwise you hide all kinds of other errors.

Comment (asalkeld) [reply pkg/templates/contents.go]: probably this error: https://github.com/hashicorp/go-getter/blob/a2ebce998f8d4105bd4b78d6c99a12803ad97a45/get_git.go#L51

**Review by asalkeld — APPROVED** [review]

**Pull request merged**